### ISSUE TITLE
Issue mentions from dotcom not carried through to inline comments

### ISSUE THREAD

**Issue by tierninho** [github/VisualStudio#1180]
If a user references another issue in a comment on dotcom, the hyperlinked text does not carry through to inline comments. 

(Btw, not expecting the reserve scenario to work)

<img width="716" alt="screen shot 2017-08-21 at 11 39 10 am" src="https://user-images.githubusercontent.com/14828183/29527185-1ef097ce-8666-11e7-9a6a-4d18a6e0039c.png">

Cannot click the "pound-sign-1":
<img width="286" alt="screen shot 2017-08-21 at 11 44 51 am" src="https://user-images.githubusercontent.com/14828183/29527205-2eb71c0a-8666-11e7-9ab6-b9fea6dfce8d.png">

**Comment by grokys** [comment]
Yeah, this is similar to #1164 in that we need to extend out markdown parser to handle issue links.

**Comment by grokys** [comment]
Removed the "bug" label from this as it's expected behavior because we don't have this feature yet.
